src/app/shared/services/record.service.ts

```diff
@@ -41,4 +45,12 @@ export class RecordService {
       .map(res => res.json());
   }
 
+  saveRecord(record: Object): Observable<Object> {
+    let postData = {
+      'pid_type': this.type,
+      'recid': this.recid,
+      'json_data': record
+    };
+    return this.http.post('http://localhost:5000/editor/save', postData).map(res => res.json());
```

Comment:
I can define then something like a const prefix inside the service which is shared. And after to change all the urls in relation to this const.